sdk/identity/azure-identity/test/perf/inc/azure/identity/test/secret_credential_test.hpp

```diff
@@ -76,7 +80,8 @@ namespace Azure { namespace Identity { namespace Test {
           {"TenantId", {"--tenantId"}, "The tenant Id for the authentication.", 1, true},
           {"ClientId", {"--clientId"}, "The client Id for the authentication.", 1, true},
           {"Secret", {"--secret"}, "The secret for authentication.", 1, true, true},
-          {"Scope", {"--scope"}, "One scope to request access to.", 1, true}};
+          {"Scope", {"--scope"}, "One scope to request access to.", 1, true},
+          {"Cache", {"--cache"}, "Use credential cache.", 1, false}};
```

Comment:
I see that in options.cpp, you rearranged switches alphabetically, but here you are putting C after S. I think it should be the either one. Either keep them historically (smaller diff), or alphabetically